src/react-components/src/components/00-base/Container/Modal.tsx

```diff
@@ -0,0 +1,32 @@
+/* Full-screen modal that appears on top of the body */
+import * as React from "react";
+import bem from "../../../utils/bem";
+
+export interface ModalProps {
+  /** Used by BEM CSS */
+  modifiers?: string[];
+
+  /** Used by BEM CSS */
+  blockName?: string;
+}
+
+export default class Modal extends React.Component<ModalProps, {}> {
```

Comment:
This is nice but there are no tests or stories for this (even if it's a small component).